src/landing-pages/shopify/index.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import "./index.scss";
+import imglaptop from "./images/imgfooterHeader.png";
+import logo from "./images/logo.jpg";
+import { ReactComponent as Logoshop } from "./images/logoshopify.svg";
+import { ReactComponent as Icon2 } from "./images/svgexport-2.svg";
+import { ReactComponent as Icon3 } from "./images/svgexport-3.svg";
+import { ReactComponent as Icon4 } from "./images/svgexport-4.svg";
```

Comment:
rename the name of icon files with meaningful names, don't name it Icon1, Icon2, Icon3. Ask

Reply:
svgexport1,2,3 are meaningless